<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoundedRectGenDlg</class>
 <widget class="QWidget" name="RoundedRectGenDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X弯道半径：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y弯道半径：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_XR">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_YR">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>边框</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_LineType">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L0.ico</normaloff>:/lineType/images/LineType/L0.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L1.ico</normaloff>:/lineType/images/LineType/L1.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L2.ico</normaloff>:/lineType/images/LineType/L2.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L3.ico</normaloff>:/lineType/images/LineType/L3.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L4.ico</normaloff>:/lineType/images/LineType/L4.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L5.ico</normaloff>:/lineType/images/LineType/L5.ico</iconset>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>线宽度：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_LineWidth">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>线颜色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_LineColorBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>321</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>填充</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>样式：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_Pattern">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>背景色：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>前景色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_BackColorBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_FrontColorBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_Alpha">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>透明度：</string>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="spinbox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID：</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
